apache/superset
# refactor: Renders Explore in SPA

### SUMMARY
Renders Explore in SPA under `/explore`. Previously, Explore was displayed under `/superset/explore`. This will open a number of possibilities for in-context features and also reduce the development complexity associated with keeping multiple React applications.

This PR also replaces SPA-related Explore navigation with React Router and removes the previous entry point files.

Follow-up of https://github.com/apache/superset/pull/20519

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
https://user-images.githubusercontent.com/70410625/176955027-010c15b7-d3a7-4537-86ef-c8d7e40a6384.mov

### TESTING INSTRUCTIONS
We need to test all features that are associated with Explore, like:
- Navigate from multiple places to Explore (welcome page, charts list, add a chart, dashboard, etc)
- Refresh the Explore page and check if the content persists
- Check permalinks
- Check Explore save flow

### ADDITIONAL INFORMATION
